packages/riipen-ui/src/components/ListItem.test.jsx

```diff
@@ -0,0 +1,182 @@
+import { mount } from "enzyme";
+import toJson from "enzyme-to-json";
+import React from "react";
+
+import ListItem from "./ListItem";
+
+describe("<ListItem>", () => {
+  it("renders without errors", () => {
+    let error;
+
+    try {
+      mount(<ListItem />);
+    } catch (e) {
+      error = e;
+    }
+
+    expect(error).toEqual(undefined);
+  });
+
+  it("renders correct snapshot", () => {
+    const wrapper = mount(<ListItem />);
+
+    expect(toJson(wrapper)).toMatchSnapshot();
+  });
+
+  describe("default props", () => {
+    it("sets correct default props", () => {
+      const wrapper = mount(<ListItem />);
+
+      const component = wrapper.find("ListItem");
+      expect(component.props().color).toEqual("primary");
+      expect(component.props().classes).toEqual([]);
+    });
+  });
+
+  describe("children prop", () => {
+    it("renders given children", () => {
+      const child = <div>childreeennn</div>;
+
+      const wrapper = mount(<ListItem>{child}</ListItem>);
+
+      expect(
+        wrapper
+          .find("ListItem")
+          .childAt(0)
+          .contains(child)
+      ).toEqual(true);
+    });
+  });
+
+  describe("classes prop", () => {
+    it("applies given classes", () => {
+      const classes = ["classOne"];
+
+      const wrapper = mount(<ListItem classes={classes} />);
+
+      expect(
+        wrapper
+          .find("ListItem")
+          .childAt(0)
+          .hasClass(classes[0])
+      ).toEqual(true);
+    });
+  });
+
+  describe("color prop", () => {
+    it("applies valid custom color", () => {
+      const color = "secondary";
+
+      const wrapper = mount(<ListItem color={color} />);
+
+      expect(
+        wrapper
+          .find("ListItem")
+          .childAt(0)
+          .hasClass(color)
```

Comment:
Am I crazy, or does the color class actually do nothing. Check out the component source code. There isn't actually any `.primary` or `.secondary` CSS classes. It seems like all it is really doing is affecting the `focusVisible` color. 

I think we should update this test to actually test that , or maybe move this test to be part of the focus tests down below.

Reply:
Since the CSS uses selectors (.focusVisible), is there no way to test this prop? Because I can't move it to inline style?

Reply:
Ya that is super annoying. I always hate saying "this code is untestable" because usually that's a sign of bad code, but I also cannot think of any way to make this testable. So let's just maybe remove this test since it's not actually testing anything.